### PR TITLE
feat: add mutex to in band runner

### PR DESCRIPTION
**Pull request by SimenB**
Same as #5, but for #6.

### CODE REVIEW

File: lib/createJestRunner.js [hunk 1]
```diff
@@ -42,36 +42,39 @@ const createRunner = runPath => {
       onFailure,
       options,
     ) {
+      const mutex = throat(1);
```
Comment (SimenB) [inline lib/createJestRunner.js]: https://github.com/jest-community/create-jest-runner/pull/7/files?w=1 recommended

**Comment by SimenB** [comment]
Could use semantic release :D But yeah I'll update it

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,4 +23,4 @@
 
 ## 0.1.0
 
-* Initial Release
\ No newline at end of file
+* Initial Release
```
Comment (SimenB) [inline CHANGELOG.md]: no idea what's up, I edited in the GitHub UI. probably added a trailing newline

Comment (keplersj) [reply CHANGELOG.md]: IIRC it does a trailing newlines by default.

**Pull request merged**